provision/infinispan/ispn-helm/templates/infinispan.yaml

```diff
@@ -0,0 +1,92 @@
+{{- $namespace := .Values.namespace | required ".Values.namespace is required." -}}
+apiVersion: infinispan.org/v1
+kind: Infinispan
+metadata:
+  name: infinispan
+  namespace: {{ $namespace }}
+  annotations:
+    infinispan.org/monitoring: 'true'
+spec:
+  affinity:
```

Comment:
I believe these are the default values right? We can omit these and they will automatically be populated.